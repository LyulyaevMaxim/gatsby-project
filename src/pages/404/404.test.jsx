import React from 'react'
import renderer from 'react-test-renderer'
import NotFoundPage from './index.en'

describe('NotFoundPage', () => {
  it('renders correctly', () => {
    const data = {
      data: {
        site: {
          siteMetadata: {
            title: `Default Starter`,
          },
        },
      },
      pathContext: {
        langKey: 'en',
      },
    }

    const tree = renderer.create(<NotFoundPage {...data} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
