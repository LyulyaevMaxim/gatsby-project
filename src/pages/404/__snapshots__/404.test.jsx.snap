// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<button
  css={
    Object {
      "map": undefined,
      "name": "yu9yz2",
      "next": undefined,
      "styles": "
    padding: 1.5rem 1rem;
  ",
    }
  }
  type="button"
>
  Text
</button>
`;
